cypress/integration/BoardingPassesInfo.js

```diff
@@ -0,0 +1,67 @@
+// @noflow
+/* global
+  Cypress,
+  cy
+*/
+/* eslint-disable jest/valid-expect */
+
+describe('Boarding Passes Info', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('[data-cy=btn-existent-booking]').click();
```

Comment:
this looks same as in `UserCanLogIntoSmartFAQ.js`, do you think you could create a custom command for it? 🙂  You can take inspiration from FE too, e.g. their `mmbLoad`